packages/sdk-node/src/index.ts

```diff
@@ -24,10 +24,9 @@
 
  */
 
-export * from './3.1/methods'
-export * as models31 from './3.1/models'
-export * from './3.1/streams'
-export * from './4.0/methods'
-export * from './4.0/models'
-export * from './4.0/streams'
+// Version 21.0.4
+ export * from './nodeServices'
```

Comment:
minor nit but looks like a leading space?

Reply:
I'll fix that with the next PR